### DISCUSSION TITLE
Node Groups release doesn't exist anymore? BlenderMalt zips not available for download.

### DISCUSSION THREAD

**Discussion by dibli-goost** [bnpr/Malt#505] [Q&A] [answered]
@pragma37 I could have sworn there was a "release" with the compiled BlenderMalt zips for the node groups branch, located on this page: https://github.com/bnpr/Malt/releases

But it's not there anymore. Looking at the tag page, where a node-groups-latest page does exist. The zips aren't there though, just the source code. The development branch does have the zips, it's just listed here as "downloads".
![image](https://github.com/bnpr/Malt/assets/31767887/66f2710b-7085-4bc3-b78c-6f1e9b537c68)

From what I understand, there's an action that's supposed to be run which does this process automatically. It seems to have ran, and passed. But no zips were actually supplied?
![image](https://github.com/bnpr/Malt/assets/31767887/cacdf9bf-1454-4af4-bfa6-1af0ba6f3421)

**Comment by pragma37** [comment, marked as answer]
Sometimes releases get marked as a Draft, I have no idea why.
![imagen](https://github.com/bnpr/Malt/assets/36610999/1a61a556-28e6-437a-a090-e3d664c5aa0d)

It should be public now.
